android/app/src/main/kotlin/io/rebble/cobble/bridges/common/AppInstallFlutterBridge.kt

```diff
@@ -77,12 +80,20 @@ class AppInstallFlutterBridge @Inject constructor(
         }
     }
 
+
+    private fun getStreamFromUri(uri: String): InputStream? {
+        if (uri.startsWith("content"))
+            return context.contentResolver
+                    .openInputStream(Uri.parse(uri))
+        return URL(uri).openStream()
```

Comment:
We should probably use OkHttp here instead of archaic java.net.URL connection socket.

Or even better: Flutter should handle downloading into local app storage (so we also implement iOS version that way) and then pass file:// url back